stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/state/messages/MessagesState.kt

```diff
@@ -41,5 +41,5 @@ public data class MessagesState(
     val currentUser: User? = null,
     val newMessageState: NewMessageState? = null,
     val parentMessageId: String? = null,
-    val unreadCount: Int = 0,
```

Comment:
Did you do this manually or through the formatter? It should have a comma at the end when formatted

Reply:
Might have deleted it by accident, but i ran the formatter so i am not sure.